iOS/Flows/Conversation/Conversation List/ConversationListViewController.swift

```diff
@@ -45,12 +50,14 @@ class ConversationListViewController: FullScreenViewController,
 
     @Published var state: ConversationUIState = .read
 
-    private let userIDs: [UserId]
+    /// A list of conversation members used to filter conversations. We'll only show conversations with this exact set of members.
+    private let members: [ConversationMember]
+
+    init(members: [ConversationMember]) {
```

Comment:
Conversation list VC is now initialized with members. It will get all conversations that contain that exact set of members.